frontend/static/js/app.js

```diff
@@ -110,11 +110,15 @@ Vue.component("correction-view", {
       this.loadNextCorrection();
     },
     accept: function() {
+      console.log(this.reviewComment)
       this.execAction('approve', {
-      corrected_value: this.correction.val
+      corrected_value: this.correction.val,
+      comment: this.reviewComment ? this.reviewComment : null,

```

Comment:
instead of repeating the code in these two places, we can make it so that when checkbox is unchecked, we reset the comment property to null